### DISCUSSION TITLE
Should title in head link to baseurl?

### DISCUSSION THREAD

**Discussion by epogrebnyak** [thuliteio/doks#98] [Q&A] [answered]
I'm starting to use the theme on an address like `example.com/subsite`. In the upper left corner, we have a bold title `Doks` which would link to `example.com` not `example.com/subsite`

![изображение](https://user-images.githubusercontent.com/9265326/106436727-82092f00-6485-11eb-910b-525ae493d2e6.png)

I think it is controlled here:

https://github.com/h-enk/doks/blob/41d63a2528f574e3934d58fb30811ce9fb879798/layouts/partials/header/header.html#L6

Can this link direct to `baseurl`? Would it break anything if it does?

**Comment by h-enk** [comment, marked as answer]
Yes, link it like so: `{{ .Site.BaseURL | absURL }}`

Hugo does not make it easy to realize your use case...

I guess you've set `baseurl = "localhost/foo/"` (or something like that) in `.config/_default/config.toml`. And that you still have issues loading fonts and images?

See also:
- https://discourse.gohugo.io/t/run-site-from-subdirectory-in-localhost-live-web-server/11620/2
- https://discourse.gohugo.io/t/can-i-reference-baseurl-from-content/2686/3

**Reply by epogrebnyak** [reply]
Thank you, will try soon! The Hugo docs are a little ambiguous and  baseurl /subsite case - for example if there should be a slash at the end. I use actual url for the baseurl stratting with `https://`, and Hugo is smart to derive the `localhost/subsite` from that. The little inconvenience is that physically loading an `index.html` into browser does not work, but `hugo serve` works fine. Will post a link to a working example once tested.

**Reply by epogrebnyak** [reply]
The biggest pleasant surprise that changing `baseurl="https://epogrebnyak.gitlab.io/doks-howto"` as in https://gitlab.com/epogrebnyak/doks-howto/ does not break the netlify deployment which seems to ignore the subsite url somehow.

**Reply by epogrebnyak** [reply]
Also settled on `.Site.Home.RelPermalink` for this line in `doks/layouts/partials/header/header.html`:

```<a class="navbar-brand order-1 order-md-0 mr-auto" href="{{ .Site.Home.RelPermalink | absURL }}">{{ .Site.Params.Title }}</a>```

It makes clicking at localhost remain at localhost, better when testing.

**Reply by h-enk** [reply]
Nice work. I'm curious to see how the Jost* font shows in other languages.

![2021-02-01_19-46-26](https://user-images.githubusercontent.com/3902872/106503679-40e53f00-64c6-11eb-81d8-088a868622d2.png)

**Reply by epogrebnyak** [reply]
The fonts look good to me. At first I noticed a very minor observation that text in English is smaller than in Russian is same sentence, but it went away - maybe something needed to be uploaded to render properly.  

![изображение](https://user-images.githubusercontent.com/9265326/106570509-07a0e380-6547-11eb-8f0f-8d50ca290d0b.png)
